actionpack/test/dispatch/request/json_params_parsing_test.rb

```diff
@@ -37,9 +37,9 @@ def teardown
     )
   end
 
-  test "parses json params for application/vnd.api+json" do
+  test "does not parses unregistered media types such as application/vnd.api+json" do
     assert_parses(
-      {"person" => {"name" => "David"}},
+      {},
```

Comment:
I kept this test around both in case anyone is looking for json api support, and because I didn't see the behavior tested elsewhere.

However, I removed the test for rootless JSON as it duplicates what I left here and also isn't relevant to JSON API.